app/pages/dashboard/TideDashboard.js

```diff
@@ -796,6 +796,8 @@ export const TideDashboard = (props) => {
   const ldClient = useLDClient();
   const ldContext = ldClient.getContext();
 
+  const isTideDashboardEnabled = ldContext?.clinic?.tier && (clinic?.entitlements?.tideDashboard || showTideDashboard);
```

Comment:
I'm wondering about this.  Would this change still work to fix the issue?
```suggestion
  const isTideDashboardEnabled = (ldContext?.clinic?.tier && showTideDashboard) || clinic?.entitlements?.tideDashboard;
```

My thinking is that this way, if we have trouble connecting to LD, but the clinic tier provides an entitlement that enables the dashboard, we would still be able to show the dashboard.

Otherwise, in the case of LD network errors, no users could see the dashboard, even if their clinic tier (or clinic entitlements, if we eventually set these directly on the clinic objects on the backend as has been talked about) would allow for it.

Thoughs?